### ISSUE TITLE
Auto-download - bug ?

### ISSUE THREAD

**Issue by Vol-2016** [Pmmlabs/OpenPeriscope#70]
Кажется, при введенном выборе из mp4/ts - 
авто-загрузка срабатывает только для mp4.

**Comment by Pmmlabs** [comment]
То есть вручную работает а авто нет?

**Comment by Vol-2016** [comment]
Как а OpenPeriscope ручную - не знаю 
(пользуюсь только скриптами от n3tman либо rharkanson, для REPLAY так гораздо быстрее).
А вот с автоматом как сказал.

**Comment by Vol-2016** [comment]
Так, попробовал загрузить вручную с выбором .ts - и получил вот что:

"[NULL @ 00a0b600] Requested output format 'mpegts' is not a suitable output format
D:_PERISCOPE\Videos_OP\2017-01-11_08.45_zari_nz_новая зеландия. вулкан закончился. теперь можно и поговорить=).ts: Invalid argument"
  
            
 
Write

**Comment by Pmmlabs** [comment]
это потому что прилагаемый ffmpeg скомпилирован без поддержки TS. Ну на самом деле я не собираюсь данную реализацию включать в релиз, потому что для скачивание TS лучше сделать обычным скачиванием файлов с последующей склейкой. (так кстати bat скрипты и работают)
